cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1408,6 +1422,10 @@ public boolean isNotificationReceivedListenerCallback() {
         return notificationReceivedListener != null && notificationReceivedListener instanceof NotificationReceivedCallbackListener;
     }
 
+    public boolean isNotificationOpenedCallbackListenerUsed() {
```

Comment:
I think the grammar is not so good for the function name, any better idea maybe?
isNotificationOpenedCallbackListenerUsed sounds bad to me

Reply:
isNotificationOpenedCallbackListener is it okay?

Reply:
isNotificationUsingOpenedCallbackListener

Reply:
okay i will change